prive/utils/data.py

```diff
@@ -127,4 +127,38 @@ def index_split(max_index, split_size, num_splits):
     np.random.shuffle(index_array)
     indices += [index_array[split_size*i : split_size*(i+1)] for i in range(remainder_splits)]
 
-    return indices
\ No newline at end of file
+    return indices
+
+def get_dtype(representation, column_type):
+    """
+
+    Based on the json file schema from the dataset module map type/representation to pandas types.
+    See data-schema-doc.rst documentation for more details.
+
+    Parameters
+    ----------
+    representation: object
+        Representation of column based on the schema this can be a string, and integer, or a list
+    column_type: str
+        String referring to abstract type of the data column (e.g. countable/ordered, finite, etc).
+
+    Returns
+    -------
+    dtype
+        A type for the given column.
+
+    """
+    if representation == 'integer':
+        return np.int
+    elif representation == 'number':
+        return np.float
+    elif representation == 'date':
+        return pd.datetime
+    elif isinstance(representation, (list, str)):
+        ordered = True if 'ordered' in column_type else False
+        return pd.CategoricalDtype(categories=representation, ordered=ordered)
```

Comment:
Now i see tests are failing locally for me and not in the CI ...

Reply:
It seems weird that pandas would be getting confused, because surely handling types (especially types they've got in their own library) should be natural. I see the CI are passing. Do you have an example of your tests failing?

Reply:
Tests passing for me (locally) albeit with a bunch of warnings about `np.int` and `np.float` being deprecated synonyms for `int` and `float`.